sdv/metadata/metadata.py

```diff
@@ -99,3 +99,31 @@ def _convert_to_single_table(self):
             )
 
         return next(iter(self.tables.values()), SingleTableMetadata())
+
+    def set_sequence_index(self, table_name, column_name):
+        """Set the sequence index of a table.
+
+        Args:
+            table_name (str):
+                Name of the table to set the sequence index.
+            column_name (str):
+                Name of the sequence index column.
+        """
+        self._validate_table_exists(table_name)
+        if len(self.tables) > 1:
+            warnings.warn('Sequential modeling is not yet supported on SDV Multi Table models.')
+        self.tables[table_name].set_sequence_index(column_name)
+
+    def set_sequence_key(self, table_name, column_name):
+        """Set the sequence key of a table.
+
+        Args:
+            table_name (str):
+                Name of the table to set the sequence key.
+            column_name (str, tulple[str]):
+                Name (or tuple of names) of the sequence key column(s).
+        """
+        self._validate_table_exists(table_name)
+        if len(self.tables) > 1:
+            warnings.warn('Sequential modeling is not yet supported on SDV Multi Table models.')
+        self.tables[table_name].set_sequence_key(column_name)
```

Comment:
These two methods in `MultiTableMetadata` would always throw the warning when used in `PARSynthesizer`